tests/50-sd-verify.js

```diff
@@ -87,17 +87,20 @@ describe('ecdsa-sd-2023 (verify)', function() {
                 verifierSupportedEcdsaKeyTypes) {
                 if(issuerSupportedEcdsaKeyTypes.includes(
                   verifierSupportedEcdsaKeyType)) {
+                  // create initial signed Vc
```

Comment:
```suggestion
                  // create initial signed VC
```

Reply:
or maybe
```suggestion
                  // create initial signedVc
```

Reply:
@TallTed in code we're trying to avoid SHOUTY CASE EVERYWHERE, but in comments standard acronyms are fine so I think `signed VC` is what we want here. also thanks for the review so far =)